Infrastructure/config.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  servers: [
+    {
+      description: 'server',
+      name: 'aarchie',
+      domain: 'ehl.cdot.systems',
+      port: '2200'
+    },
+    {
+      description: 'server',
+      name: 'bbetty',
+      domain: 'ehl.cdot.systems',
+      port: '2202'
+    },
+    {
+      description: 'server',
+      name: 'ccharlie',
+      domain: 'ehl.cdot.systems',
+      port: '2205'
+    },
+    {
+      description: 'server',
+      name: 'xerxes',
+      domain: 'xerxes.cdot.systems',
+      port: '22'
+    }
+  ],
+  workstations: [
+    { host: 'bbb', ipAddress: 'bbb.cdot.systems' },
+    { host: 'vega', ipAddress: 'vega.cdot.systems' },
+    { host: 'rob', ipAddress: 'rob.cdot.systems' },
+    { host: 'ps', ipAddress: 'ps.cdot.systems' },
+    { host: 'italy', ipAddress: 'italy.cdot.systems' },
+    { host: 'canada', ipAddress: 'canada.cdot.systems' },
+    { host: 'narnia', ipAddress: 'narnia.cdot.systems' },
+    { host: 'spain', ipAddress: 'spain.cdot.systems' },
+    { host: 'rome', ipAddress: 'rome.cdot.systems' },
```

Comment:
these are not ipAddresses, maybe we could switch to `hosts`?

Reply:
Changed in `dummy-data.js`, `config.js`, and `checkService.js`